addons/revolutionaries/config.cpp

```diff
@@ -0,0 +1,94 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {
+			//Backpacks
+			"TACU_Revolutionaries_Backpack_Mk14",
```

Comment:
All of your entries have tabs instead of spacing, and it's incorrectly formatted.
```suggestion
            "TACU_Revolutionaries_Backpack_Mk14",
```